app/temporal/src/workflows/importer.workflow.ts

```diff
@@ -281,41 +261,37 @@ export async function importer(params: ImporterWorkflowParams) {
         });
       }
     }
+    await acts.applyPatches({ importerId: sourceFile!.bucket, patches });
```

Comment:
Müssen nochmal prüfen, wie genau wir nun die Patches umsetzen wollen. Theoretisch würde es auch gehen, wenn wir beim Update gleich applyPatch oder so aufrufen und das in der DB speichern